woocommerce/woocommerce-ios
# Add GitHub release creation to the release tooling

This PR adds a new build_release lane, based on the `build_and_upload_itc` one, that, after the build, creates the GitHub release and uploads the `xarchive`.

**To test**:
- Update `Version.public.xcconfig` to a version that doesn't have a release on GitHub.
- Checkout a new test branch from this one.
- [Optional] Comment out the `testflight` step from the `build_and_upload_itc` lane, so that we don't upload unuseful stuff.
- Commit the changes you made above to the test branch.
- Run bundle exec fastlane build_release.
- Verify that after the build, a new <version> release has been drafted in GitHub and the XArchive and the release notes have been uploaded.

_Clean Up_
- Delete the release from GitHub.
- Remove the test branch from your local repository.

Update release notes:
